include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -0,0 +1,497 @@
+    
+    /* GraphicsPipelineLibraryPropertiesEXT *//* provided by VK_EXT_graphics_pipeline_library */
+    //VkBool32           graphicsPipelineLibraryFastLinking;
+    //VkBool32           graphicsPipelineLibraryIndependentInterpolationDecoration;
+
+    /* InlineUniformBlockPropertiesEXT ---> MOVED TO Vulkan 1.3 Core  */
+
+    /* LineRasterizationPropertiesEXT *//* provided by VK_EXT_line_rasterization */
+    //uint32_t           lineSubPixelPrecisionBits;
+
+    /* MultiDrawPropertiesEXT *//* provided by VK_EXT_multi_draw */
+    //uint32_t           maxMultiDrawCount;
+
+    /* PCIBusInfoPropertiesEXT *//* provided by VK_EXT_pci_bus_info */
+    //uint32_t           pciDomain;
+    //uint32_t           pciBus;
+    //uint32_t           pciDevice;
+    //uint32_t           pciFunction;
+
+    /* DrmPropertiesEXT *//* provided by VK_EXT_physical_device_drm */
+    //VkBool32           hasPrimary;
+    //VkBool32           hasRender;
+    //int64_t            primaryMajor;
+    //int64_t            primaryMinor;
+    //int64_t            renderMajor;
+    //int64_t            renderMinor;
+
+    /* ProvokingVertexPropertiesEXT *//* provided by VK_EXT_provoking_vertex */
+    //VkBool32           provokingVertexModePerPipeline;
+    //VkBool32           transformFeedbackPreservesTriangleFanProvokingVertex;
+
+    /* Robustness2PropertiesEXT *//* provided by VK_EXT_robustness2 */
+    //VkDeviceSize       robustStorageBufferAccessSizeAlignment;
+    //VkDeviceSize       robustUniformBufferAccessSizeAlignment;
+
+    /* SamplerFilterMinmaxPropertiesEXT ---> MOVED TO Vulkan 1.2 Core  */
+
+    /* SampleLocationsPropertiesEXT *//* provided by VK_EXT_sample_locations */
+    //VkSampleCountFlags    sampleLocationSampleCounts;
+    //VkExtent2D            maxSampleLocationGridSize;
+    //float                 sampleLocationCoordinateRange[2];
+    //uint32_t              sampleLocationSubPixelBits;
+    //VkBool32              variableSampleLocations;
+
+    /* SubgroupSizeControlPropertiesEXT ---> MOVED TO Vulkan 1.3 Core  */
+
+    /* TexelBufferAlignmentPropertiesEXT ---> MOVED TO Vulkan 1.3 Core  */
+
+    /* TransformFeedbackPropertiesEXT *//* provided by VK_EXT_transform_feedback */
+    //uint32_t           maxTransformFeedbackStreams;
+    //uint32_t           maxTransformFeedbackBuffers;
+    //VkDeviceSize       maxTransformFeedbackBufferSize;
+    //uint32_t           maxTransformFeedbackStreamDataSize;
+    //uint32_t           maxTransformFeedbackBufferDataSize;
+    //uint32_t           maxTransformFeedbackBufferDataStride;
+    //VkBool32           transformFeedbackQueries;
+    //VkBool32           transformFeedbackStreamsLinesTriangles;
+    //VkBool32           transformFeedbackRasterizationStreamSelect;
+    //VkBool32           transformFeedbackDraw;
+
+    /* VertexAttributeDivisorPropertiesEXT *//* provided by VK_EXT_vertex_attribute_divisor */
+    //uint32_t           maxVertexAttribDivisor;
```

Comment:
in OpenGL its `~0u`

Reply:
but naturally a TODO because we have the `VK_EXT_vertex_attribute_divisor` as a TODO as well